rllib/env/tests/test_env_runner_group.py

```diff
@@ -91,6 +92,40 @@ def test_foreach_env_runner_async(self):
 
         ws.stop()
 
+    def test_foreach_env_runner_async_fetch_ready(self):
+        """Test to make sure basic asychronous calls to remote workers work."""
+        ws = EnvRunnerGroup(
+            config=(
+                PPOConfig()
+                .environment("CartPole-v1")
+                .env_runners(num_env_runners=2, rollout_fragment_length=1)
+            ),
+        )
+
+        # Sample from both env runners.
+        # First call to foreach_env_runner_async_fetch_ready should not return ready results.
+        self.assertEqual(
+            len(
+                ws.foreach_env_runner_async_fetch_ready(
+                    lambda w: w.sample(),
+                    tag="sample",
+                )
+            ),
+            0,
+        )
+        time.sleep(1)
```

Comment:
I also tested successfully with 0.01 seconds. So I expect this test to succeed on slower platforms, too.